BUILDING.md

```diff
@@ -1,9 +1,10 @@
-## Building Plugins with OpenSeearch
+## Building Plugins with OpenSearch
 
 Until OpenSearch and other artifacts are published to Maven Central [OpenSearch#581](https://github.com/opensearch-project/OpenSearch/issues/581), plugins may require building all their dependencies and publishing them to Maven local.
 
-- [Publish OpenSearch to Maven Local](#publish-opensearch-to-maven-local)
-- [Use OpenSearch from Maven Local in Plugins](#use-opensearch-from-maven-local-in-plugins)
+- [Building Plugins with OpenSearch](#building-plugins-with-opensearch)
```

Comment:
Do we need adding this section? as this content is already part of it?
If you'd like to have a tree, probably we can put this at the top just like https://github.com/opensearch-project/opensearch-plugins/blob/main/README.md